### DISCUSSION TITLE
YouTube ReVanced "Custom Branding" Patch (and similar) Setup

### DISCUSSION THREAD

**Discussion by Kitsune-Ultima** [ReVanced/revanced-discussions#1090] [Question] [answered]
Simply, I haven't found any documentation or examples regarding the "Custom Branding" patch such as:

- Minimum/Maximum Icon Resolution for each specified folder and required image (if different),
- Minimum/Maximum Icon Filesize for each specified folder and required image (if different),
- Accepted Icon Bitdepths for each specified folder and required image (if different)

The ReVanced Manager shows how to layout the folder structure and what filenames are needed but neglects to specify any other requirements.

**Comment by oSumAtrIX** [comment]
It doesn't matter

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Bit depth, encoding type, and file size do not matter at all.

Resolution doesn't matter much since the device will resize to fit (although it might use nearest neighbor or some other poor resizing strategy).  If you want the ideal resolution then use the same resolution as the files found [here](https://github.com/ReVanced/revanced-patches/tree/v5.0.0/patches/src/main/resources/custom-branding)
